src/main/java/org/example/service/TableService.java

```diff
@@ -0,0 +1,56 @@
+package org.example.service;
+
+import org.example.model.Developer;
+
+import java.util.List;
+
+public class TableService {
+    private static TableService INSTANCE;
```

Comment:
Check the naming conventions.
When the variable title should be written in caps.

Reply:
fixed